model/src/main/kotlin/FileFormat.kt

```diff
@@ -83,6 +83,19 @@ fun File.mapper() = FileFormat.forFile(this).mapper
  */
 inline fun <reified T : Any> File.readValue(): T = mapper().readValue(this)
 
+/**
+ * Use the Jackson mapper returned from [File.mapper] to read an object of type [T] from this file, or return the
+ * [default] value if the file has no content.
+ */
+inline fun <reified T : Any> File.readValueOrDefault(default: T): T =
+    runCatching { readValue<T>() }.getOrDefault(default)
```

Comment:
If I understand correctly, every exception is now caught, ignored, and the default value is returned. If so, is this really what we want? What about non-empty files that are syntactically incorrect? Think about a user who missed a bracket or something and now wonders why his configuration changes have no effect.

Reply:
Doh, you're right. Now I recall why I did it differently to begin with. However, my original `readTree` / `treeToValue` combination seems to trigger a Kotlin compiler bug :-(